BiCo/src/app/components/negocios/mis-negocios/mis-negocios.component.ts

```diff
@@ -0,0 +1,32 @@
+import { Component, OnInit } from '@angular/core';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/ISPP-2021/BiCo-Frontend/pullRequest?prid=7237503)
```suggestion
import { Component, OnInit } from "@angular/core";
```